versioned_docs/version-8.7/components/best-practices/architecture/sizing-your-environment.md

```diff
@@ -150,11 +150,11 @@ Contact your Customer Success Manager to increase the cluster size beyond the ma
 :::
 
 | Cluster size                                                                        |                                  1x |                                  2x |                               3x |                               4x |
-| :---------------------------------------------------------------------------------- | ----------------------------------: | ----------------------------------: | -------------------------------: | -------------------------------: |
+| :---------------------------------------------------------------------------------- |------------------------------------:|------------------------------------:|---------------------------------:|---------------------------------:|
 | Max Throughput **Tasks/day** **\***                                                 |                                 9 M |                                18 M |                             27 M |                             36 M |
 | Max Throughput **Tasks/second** **\***                                              |                                 100 |                                 200 |                              300 |                              400 |
 | Max Throughput **Process Instances/second** **\*\***                                |                                   5 |                                  10 |                               15 |                               20 |
-| Max Total Number of Process Instances stored (in Elasticsearch in total) **\*\*\*** |                                75 k |                               150 k |                            225 k |                            300 k |
+| Max Total Number of Process Instances stored (in Elasticsearch in total) **\*\*\*** |                               200 k |                               400 k |                            600 k |                            800 k |
 | Approximate resources provisioned **\*\*\*\***                                      | 11 vCPU, 22 GB memory, 192 GB disk. | 22 vCPU, 44 GB memory, 384 GB disk. | 33 vCPU, 66 GB mem, 576 GB disk. | 44 vCPU, 88 GB mem, 768 GB disk. |
 
 The numbers in the table were measured using Camunda 8 (version 8.6), [the benchmark project](https://github.com/camunda-community-hub/camunda-8-benchmark) running on its own Kubernetes Cluster, and using a [realistic process](https://github.com/camunda/camunda/blob/main/zeebe/benchmarks/project/src/main/resources/bpmn/realistic/bankCustomerComplaintDisputeHandling.bpmn) containing a mix of BPMN symbols such as tasks, events and call activities including subprocesses. To calculate day-based metrics, an equal distribution over 24 hours is assumed.
```

Comment:
:question: why is this in folder `version-8.7`?
And why does it mention `Camunda 8 (version 8.6)` here?

Reply:
Probably the 8.6 was copy pasted at some point. I would just remove the version to void these discrepancies. Though I will point out, this was not measured using that benchmark project 😄 

And agree with Carlo, our new numbers should be in the 8.8 and docs folder, not in older versions.